allenact/base_abstractions/sensor.py

```diff
@@ -268,254 +332,3 @@ def get_observation(
-            [[[0.485, 0.456, 0.406]]], dtype=np.float32
-        ),
-        stdev: Optional[np.ndarray] = np.array(
-            [[[0.229, 0.224, 0.225]]], dtype=np.float32
-        ),
-        height: Optional[int] = None,
-        width: Optional[int] = None,
-        uuid: str = "rgb",
-        output_shape: Optional[Tuple[int, ...]] = None,
-        output_channels: int = 3,
-        unnormalized_infimum: float = 0.0,
-        unnormalized_supremum: float = 1.0,
-        scale_first: bool = True,
-        **kwargs: Any
-    ):
-        """Initializer.
-
-        # Parameters
-
-        config : If `config["use_resnet_normalization"]` is `True` then the RGB images will be normalized
-            with means `[0.485, 0.456, 0.406]` and standard deviations `[0.229, 0.224, 0.225]` (i.e. using the standard
-            resnet normalization). If both `config["height"]` and `config["width"]` are non-negative integers then
-            the RGB image returned from the environment will be rescaled to have shape
-            (config["height"], config["width"], 3) using bilinear sampling.
-        args : Extra args. Currently unused.
-        kwargs : Extra kwargs. Currently unused.
-        """
-
-        if not use_resnet_normalization:
-            mean, stdev = None, None
-
-        super().__init__(**prepare_locals_for_super(locals()))
-
-
-class DepthSensor(VisionSensor[EnvType, SubTaskType], ABC):
-    def __init__(
-        self,
-        use_normalization: bool = False,
-        mean: Optional[np.ndarray] = np.array([[0.5]], dtype=np.float32),
-        stdev: Optional[np.ndarray] = np.array([[0.25]], dtype=np.float32),
-        height: Optional[int] = None,
-        width: Optional[int] = None,
-        uuid: str = "depth",
-        output_shape: Optional[Tuple[int, ...]] = None,
-        output_channels: int = 1,
-        unnormalized_infimum: float = 0.0,
-        unnormalized_supremum: float = 5.0,
-        scale_first: bool = True,
-        **kwargs: Any
-    ):
-        """Initializer.
-
-        # Parameters
-
-        config : If `config["use_normalization"]` is `True` then the depth images will be normalized
-            with mean 0.5 and standard deviation 0.25. If both `config["height"]` and `config["width"]` are
-            non-negative integers then the depth image returned from the environment will be rescaled to have shape
-            (config["height"], config["width"]) using bilinear sampling.
-        args : Extra args. Currently unused.
-        kwargs : Extra kwargs. Currently unused.
-        """
-
-        if not use_normalization:
-            mean, stdev = None, None
-
-        super().__init__(**prepare_locals_for_super(locals()))
-
-    def get_observation(  # type: ignore
-        self, env: EnvType, task: Optional[SubTaskType], *args: Any, **kwargs: Any
-    ) -> Any:
-        depth = super().get_observation(env, task, *args, **kwargs)
-        depth = np.expand_dims(depth, 2)
-
-        return depth
```

Comment:
Can we leave some "tombstones" that point people to where these implementations now are? I.e.
```
class DepthSensor():
    def __init__(self, *args: Any, **kwargs: Any):
        raise ImportError(
            "`allenact.base_abstractions.sensor.DepthSensor` has moved!"
            " Please import ... instead."
        )
```